ai_genomics/pipeline/openalex/works_pipeline.py

```diff
@@ -0,0 +1,147 @@
+"""
+works pipeline
+--------------
+
+A pipeline that takes a list of concept IDs and years, and outputs OpenAlex API results.
+
+The thought behind this is to break the results into manageable yearly chunks. For a given year
+and high level concept, the output works may be well over 2GB in size when saved to json.
+"""
+import itertools
+import json
+import requests
+from metaflow import FlowSpec, S3, step, Parameter, retry, batch
+
+# Amend this to your desired concepts/years. OpenAlex allows up to 50 parameters
+# per query, so code is included by default to chunk up the concepts into 40s.
+CONCEPT_IDS = [
+    "C200594392",
+    "C154945302",
+    "C54355233"
+]
+
+YEARS = [
+    2022,
+    2021,
+]
+
+API_ROOT = "https://api.openalex.org/works?filter="
+
+def generate_queries(concepts, years):
+    """Generates a list of queries for the list of concepts and
+    years required.
+
+    Args:
+        concepts : list of concepts to be queried
+        years : list of years to be queried
+
+    Returns:
+        query_list : list of all queries
+    """
+    concepts_joined = "|".join(concepts)
+    query_list = []
+    for year in years:
+        query_list.append(f'{concepts_joined},publication_year:{year}')
+    return query_list
```

Comment:
This can be simplified to:

```suggestion
return [f'{concepts_joined},publication_year:{year}' for year in years]
```